src/singletons/NativeMessaging.cpp

```diff
@@ -169,18 +216,18 @@ NativeMessagingServer::~NativeMessagingServer()
     {
         qCWarning(chatterinoNativeMessage) << "Failed to remove message queue";
     }
-    this->thread.requestInterruption();
-    this->thread.quit();
+    this->thread->requestInterruption();
+    this->thread->quit();
     // Most likely, the receiver thread will still wait for a message
-    if (!this->thread.wait(250))
+    if (!this->thread->wait(250))
     {
-        this->thread.terminate();
+        this->thread->terminate();
     }
```

Comment:
Seems like we're supposed to wait after calling terminate too https://doc.qt.io/qt-6/qthread.html#terminate

Maybe something like this instead - reducing the max timeout & waiting after the call to terminate too - 
can't do a suggestion because it spans a deleted line or something
```c++
    if (!this->thread->wait(100))
    {
        this->thread->terminate();

        if (!this->thread->wait(100))
        {
            qCWarning(chatterinoNativeMessage)
                << "Failed to terminate thread cleanly";
        }
    }
```